dvc/remote/ssh/__init__.py

```diff
@@ -43,7 +46,12 @@ def __init__(self, repo, config):
             or parsed.username
             or getpass.getuser()
         )
-        self.prefix = parsed.path or "/"
+        # In Windows, there are too much dirs and file will be fetch
```

Comment:
So which test failed? I don't think data_cloud test tries to walk root. If it does then it is a bug. And we shouldn't use such hack in the RemoteSSH itself.

Reply:
You are right. This is a bug from urlparse in windows. It always return '' for `path`. Then it leaded to root directory. In Windows root directory (assume it is `C:` path), there are some directories are protected by the OS.
But please disregard this point as I fixed the urlparse on Windows in next patch already.